src/probability/probability_mass_function/basic.lean

```diff
@@ -103,6 +106,17 @@ variables (p : pmf α) (s t : set α)
 lemma to_outer_measure_apply : p.to_outer_measure s = ∑' x, s.indicator p x :=
 tsum_congr (λ x, smul_dirac_apply (p x) x s)
 
+@[simp] lemma to_outer_measure_caratheodory : p.to_outer_measure.caratheodory = ⊤ :=
+begin
+  refine (eq_top_iff.2 $ le_trans (le_Inf $ λ x hx, _) (le_sum_caratheodory _)),
+  exact let ⟨y, hy⟩ := hx in ((le_of_eq (dirac_caratheodory y).symm).trans
+    (le_smul_caratheodory _ _)).trans (le_of_eq hy),
+end
+
+lemma measurable_set_to_outer_measure (s : set α) :
+  measurable_set[p.to_outer_measure.caratheodory] s :=
+p.to_outer_measure_caratheodory.symm ▸ measurable_space.measurable_set_top
```

Comment:
Are you sure we need this lemma? It's a straightforward `rw` away from the much more reasonable `measurable_set_top`.

Reply:
I don't think so, the original use case I had for it didn't end up being necessary anyways.